api/hooks/discordBot/commands/sdtd/status.js

```diff
@@ -53,17 +53,27 @@ class Status extends Commando.Command {
       onlinePlayersStringList += `${he.decode(player.name)}, `;
     });
 
-
-    let nextHorde = (Math.trunc(serverInfo.stats.gametime.days / 7) + 1) * 7;
+    let bloodMoonDay;
+    let nextHorde;
+
+    try{
+      bloodMoonDay = await sails.helpers.sdtdApi.executeConsoleCommand(
+        SdtdServer.getAPIConfig(sdtdServer),
+        `ggs BloodMoonDay`
+      );
+      nextHorde = parseInt(bloodMoonDay.result.split(` = `)[1]);
```

Comment:
minor suggestion, while not required, you should always pass in a radix into parse int, since by default it tries to get, if for example it starts with 0, it can try to parse it as hex., forcing it to 10 means use base 10 which is normal numbers

```suggestion
       nextHorde = parseInt(bloodMoonDay.result.split(` = `)[1], 10);
```